src/main/java/org/apache/maven/plugins/javadoc/AbstractFixJavadocMojo.java

```diff
@@ -808,20 +808,6 @@ private void parseClirrTextOutputFile( File clirrTextOutputFile )
                 switch ( code )
                 {
                     case 7011:

```

Comment:
fallthrough is dangerous; perhpas extrat a helper method instead

Reply:
> fallthrough is dangerous; perhpas extrat a helper method instead

@elharo OK.

Reply:
@elharo done.